### ISSUE TITLE
Integrate doctests into datasets module

### ISSUE THREAD

**Issue by kevinsantana11** [Cloud-Drift/clouddrift#394]

**Comment by philippemiron** [comment]
FYI on this one. The yomaha dataset (and maybe others!) is pulling real-time data. If I understand correctly, this would break the `datasets.py` doctest with:

```
    >>> from clouddrift.datasets import yomaha
    >>> ds = yomaha()
    >>> ds
    <xarray.Dataset>
    Dimensions:     (obs: 1926743, traj: 12196)
    Coordinates:
        time_d      (obs) datetime64[ns] ...
        time_s      (obs) datetime64[ns] ...
        time_lp     (obs) datetime64[ns] ...
        time_lc     (obs) datetime64[ns] ...
        id          (traj) int64 ...
    Dimensions without coordinates: obs, traj
    ...
    ...
    ...
``` 

since it's just comparing the string results with what we have in the docstring.

**Comment by kevinsantana11** [comment]
> FYI on this one. The yomaha dataset (and maybe others!) is pulling real-time data. If I understand correctly, this would break the `datasets.py` doctest with:
> 
> ```
>     >>> from clouddrift.datasets import yomaha
>     >>> ds = yomaha()
>     >>> ds
>     <xarray.Dataset>
>     Dimensions:     (obs: 1926743, traj: 12196)
>     Coordinates:
>         time_d      (obs) datetime64[ns] ...
>         time_s      (obs) datetime64[ns] ...
>         time_lp     (obs) datetime64[ns] ...
>         time_lc     (obs) datetime64[ns] ...
>         id          (traj) int64 ...
>     Dimensions without coordinates: obs, traj
>     ...
>     ...
>     ...
> ```
> 
> since it's just comparing the string results with what we have in the docstring.

Good point, should have used ellipses in place of those numbers but can be addressed in a future issue/PR
